app/controllers/auth/AuthAction.scala

```diff
@@ -132,14 +131,7 @@ class AuthActionImpl @Inject() (
             result        <- block(authenticatedRequest)
             updatedResult <- sessionAuditor.auditOnce(authenticatedRequest, result)
           } yield updatedResult
-
-          (configDecorator.singleAccountEnrolmentFeature, businessHours.isTrue(LocalDateTime.now())) match {
-            case (true, true) =>
-              if (enrolmentsHelper.singleAccountEnrolmentPresent(enrolments)) updatedResult
-              else Future.successful(Redirect(configDecorator.taxEnrolmentDeniedRedirect(request.uri)))
-            case _            => updatedResult
-          }
-
+          updatedResult
```

Comment:
You can return just the for/yield rather than making it a lazy val and returning it after

Reply:
Done